pkg/testutils/mutations/mutations.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package mutations
+
+import (
+	"math/rand"
+	"strings"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/parser"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+)
+
+type StatementMutator func(stmt tree.Statement) (changed bool)
```

Comment:
We might want a different interface to this -- the thing I wrote won't change the create table statement that comes in, but instead outputs some more sql statements

Reply:
I agree. I have another PR that adds random foreign key relationships that also needs to output more statements. Once that and this PR merge I'll refactor this package to add that ability. The API here will change a lot as we add these and teach it to do what we need.